<?xml version="1.0" ?>
<robot name="ship" xmlns:xacro="http://ros.org/wiki/xacro">

    <link name="ship_base_link">
        <visual>
            <geometry>
                <mesh filename="package://underwater_sim/models/meshes/ship/ship.dae" scale="0.0025 0.0025 0.0025"/>
            </geometry>
        </visual>

        <collision name="base_link_collision">
            <geometry>
                <mesh filename="package://underwater_sim/models/meshes/ship/ship.dae"/>
            </geometry>
        </collision>
    
        <inertial>
            <mass value="95.78"/>
            <inertia ixx="57.28" ixy="0.00" ixz="-0.23" iyy="57.28" iyz="0.00" izz="6.09"/>
        </inertial>
    </link>
    
    <link name="dummy_link"/>
    
    <joint name="dummy_joint" type="fixed">
        <origin xyz="0 0 0"/>
        <parent link="ship_base_link"/>
        <child link="dummy_link"/>
    </joint>

</robot>

